niworkflows/viz/plots.py

```diff
@@ -24,33 +25,48 @@
 DINA4_LANDSCAPE = (11.69, 8.27)
 
 
-class fMRIPlot(object):
+class fMRIPlot:
     """
     Generates the fMRI Summary Plot
     """
-    __slots__ = ['func_file', 'mask_data',
-                 'tr', 'seg_data', 'confounds', 'spikes']
+    __slots__ = ('func_file', 'mask_data', 'tr', 'seg_data', 'confounds', 'spikes', '_cifti')
+
+    def __init__(
```

Comment:
Are there any changes to the function's signature or this is just black's reformatting?

Reply:
just reformatting